test/weak_convergence/weak_strat.jl

```diff
@@ -108,6 +110,27 @@ m = log(errors[end]/errors[1])/log(dts[end]/dts[1])
 
 println("RS2:", m)
 
+dts = 1 .//2 .^(5:-1:0)
+numtraj = Int(1e5)
+seed = 100
+Random.seed!(seed)
+seeds = rand(UInt, numtraj)
+
+prob = SDEProblem(f,g,u₀,tspan, p)
+ensemble_prob = EnsembleProblem(prob;
+        output_func = (sol,i) -> (h1(sol[end]),false),
+        prob_func = prob_func
+        )
+
+
+_solutions = @time generate_weak_solutions(ensemble_prob, NON(), dts, numtraj, ensemblealg=EnsembleThreads())
+
+errors = [LinearAlgebra.norm(Statistics.mean(sol.u) .- u₀.*exp(1.0*(p[1]+0.5*p[2]^2))) for sol in _solutions]
+m = log(errors[end]/errors[1])/log(dts[end]/dts[1])
+@test -(m-2) < 0.3
```

Comment:
these tests should really be on the absolute value

Reply:
Isn't norm the same as the absolute value for scalars/1d arrays https://docs.julialang.org/en/v1/stdlib/LinearAlgebra/#LinearAlgebra.norm ?

Reply:
First though, is there a reason why this file doesn't use the standard convergence simulation? It's doing the same calculation. But specifically right here, this doesn't capture both directions, i.e. `abs(m-2) < 0.3`. We can probably finish up all of the weak methods first and then turn around and make this use the standard tools. It's not a big deal, but it does make maintenance easier down the road.

Reply:
IIRC the problem was that much more memory was used in the standard tools (potentially because more than the weak final error was tested?) and then it always filled up my swap. 

Yes  (by tweaking the used random seed or using more trajectories) we could use  `abs(m-2) < 0.3` ..  at the moment because of the smallest dt value, we sometimes get a slightly better convergence.